RELEASE.md

```diff
@@ -62,44 +62,31 @@ $ git push --follow-tags
 
 ## Pre-releases
 
-### Automatic (recommended)
+to create betas 
 
-> [!TIP]
-> For detailed steps read [`beta-mode.sh`](./scripts/beta-mode.sh).
+from master checkout a new branch and run 
 
-1. Run `yarn beta-enter`
+`yarn changesets pre enter beta`
 
-   This will enter the "pre mode" of changesets, create a `prerelease/beta` branch, and push
-   it to `origin` (GitHub).
+then commit, push and open a pr
```

Comment:
commit all the files? I assume so, but imho worth mentioning to be sure that your working directory is clean and free of any changes